osu.Game/Overlays/WikiOverlay.cs

```diff
@@ -135,6 +152,7 @@ private void onPathChanged(ValueChangedEvent<string> e)
         private void onSuccess(APIWikiPage response)
         {
             wikiData.Value = response;
+            Header.LanguageDropdown.UpdateDropDown(response);
```

Comment:
I'd rather the two parameters are passed through rather than the whole response.